### PR TITLE
Changing the thief's objective to stealing ID cards.

### PR DESCRIPTION
**Pull request by kosticia**
## About the PR
Currently, the thief has an unreasonably difficult goal, especially during lowpop. This can require stealing a huge number of ID cards, namely 15. Additionally, this goal has an extremely easy difficulty for the game itself. With my change, I am proposing to change the maximum number of required IDs from 15 to 10.

## Why / Balance
This objective may require literally stealing ID cards from every second person during low-pop. This is just so much difficult, given the presence of other objectives of the thief.

## Technical details
The maximum number of required cards has been changed from 15 to 10. The difficulty of the objective itself has been increased from 0.7 to 1.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl:
- tweak: The maximum number of ID cards required to complete a thief's objective has been changed from 15 to 10.

### CODE REVIEW

**Comment by Cojoke-dot** [comment]
It's not really that difficult in comparison. The figurine one is the same but you can't steal them from people, you just have to get lucky and find them around the station or buy them. With the id you at least know where they are and can still buy more. Top that with objectives like steal 5 stamps or door remotes? Its current balance is pretty on par with other objectives.

**Comment by deltanedas** [comment]
figurine is literally just buying them, every single round theres no way you can get 18 figurines otherwise

**Comment by Cojoke-dot** [comment]
> figurine is literally just buying them, every single round theres no way you can get 18 figurines otherwise

You can buy id too. Figurines are just worse because you're not stealing them, you're not being a thief. Stealing is the whole point of the antag, stealing a ton of things(ids) is the whole point.

**Comment by kosticia** [comment]
> The figurine one is the same but you can't steal them from people, you just have to get lucky and find them around the station or buy them. With the id you at least know where they are and can still buy more.

I think the figures are the problem with the maps. On a conventional Train, this goal is quite fun to accomplish.
>Top that with objectives like steal 5 stamps or door remotes? Its current balance is pretty on par with other objectives.

 I have never seen any problems with stamps and remote remotes from airlocks, there were always enough of them

File: Resources/Prototypes/Objectives/thief.yml [hunk 1]
```diff
@@ -129,10 +129,10 @@
   - type: StealCondition
     stealGroup: IDCard
     minCollectionSize: 5
-    maxCollectionSize: 15
+    maxCollectionSize: 10
     verifyMapExistence: false
   - type: Objective
-    difficulty: 0.7
+    difficulty: 1
```
Comment (slarticodefast) [inline Resources/Prototypes/Objectives/thief.yml:135]:
```suggestion
    difficulty: 0.7
```
The number reduction is good, but keep the old difficulty, please.

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Review by slarticodefast — APPROVED** [review]

**Comment by slarticodefast** [comment]
I went ahead and adjusted the objective difficulty for you. Thank you for your contribution.

**Pull request merged**